src/style.css

```diff
@@ -1,3 +1,64 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
You should provide a fallback font-family in case the main one doesn't work. For example, you can use a generic font family like `sans-serif` as a fallback.